app/assets/javascripts/sufia/tabs.js

```diff
@@ -16,7 +17,11 @@ Blacklight.onLoad(function () {
     if (activeTab.length) {
       activeTab.tab('show');
     } else {
-      $('.nav-tabs a:first').tab('show');
+      var firstTab = $('.nav-tabs a:first');
+      // select the first tab if it has an id and is expected to be selected
+      if (firstTab[0].id != ""){
```

Comment:
Why do we care if it has an id? Can we just check that the typeof is not undefined?